client/src/model/data_layer.ts

```diff
@@ -61,8 +63,10 @@ export interface FeatureProperty {
   label: string;
   // Name of this feature property.
   name: string;
-  /// Type of data to configure parsing
+  // Type of data to configure parsing
   dataType: FeaturePropertyDataType;
+  // Whether this property is optional. Optional values may be null or not exist in the list.
+  optional?: boolean;
```

Comment:
What is the difference between something being optional and something not having a value?

Reply:
I was thinking optional would mean we wouldn't display empty / default value if it isn't present. I mainly wanted this so that I didn't have to differentiate between popup content for different agg levels - instead just mark those optional and if they aren't present don't show them. 

Thoughts?